pyMBE.py

```diff
@@ -1193,31 +1211,62 @@ def define_molecule(self, name, residue_list):
         self.df.at [index,('residue_list','')] = residue_list
         return
 
-    def define_particle(self, name, q=0, diameter=None, acidity='inert', epsilon=None, pka=None):
+    def define_particle(self, name, q=0, acidity='inert', pka=None, sigma=None, epsilon=None, cutoff=None, offset=None):
```

Comment:
In the docs says: `cutoff` defaults to `2**(1./6.) reduced_length` but in the function is None.